src/layouts/EditContactUs.jsx

```diff
@@ -757,21 +719,11 @@ const EditContactUs = ({ match }) => {
       if (!filteredFrontMatter.locations.length)
         delete filteredFrontMatter.locations
 
-      const content = concatFrontMatterMdBody(filteredFrontMatter, "")
-
-      const frontMatterParams = {
-        content,
-        sha: frontMatterSha,
-      }
-
       if (JSON.stringify(originalFrontMatter) !== JSON.stringify(frontMatter)) {
-        await axios.post(
-          `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/pages/contact-us.md`,
-          frontMatterParams,
-          {
-            withCredentials: true,
-          }
-        )
+        await saveContactUs({
+          frontMatter: filteredFrontMatter,
+          sha: frontMatterSha,
+        })
```

Comment:
This is the format expected by the endpoint currently - we can actually remove it for the homepage since the page body is not useful in this case, but our backend would end up adding it anyway because of the expected format in github. For now I'm in favour of leaving it as is just to get sanitisation working 

Removing `withCredentials` works fine on staging!